terraform/environments/production/relays.tf

```diff
@@ -47,49 +47,56 @@ module "relays" {
       cidr_range = "10.134.0.0/24"
       type       = "f1-micro"
       replicas   = 1
-      zones      = ["southamerica-east1-b"]
+      zones      = ["southamerica-east1-a", "southamerica-east1-b", "southamerica-east1-c"]
     }
 
     "us-central1" = {
       cidr_range = "10.135.0.0/24"
       type       = "f1-micro"
       replicas   = 1
-      zones      = ["us-central1-b"]
+      zones      = ["us-central1-a", "us-central1-b", "us-central1-c", "us-central1-d", "us-central1-f"]
     }
 
     "us-east1" = {
       cidr_range = "10.136.0.0/24"
       type       = "f1-micro"
-      replicas   = 2
-      zones      = ["us-east1-d"]
+      replicas   = 1
+      zones      = ["us-east1-a", "us-east1-b", "us-east1-c", "us-east1-d"]
     }
 
     "us-west2" = {
       cidr_range = "10.137.0.0/24"
-      type       = "f1-micro"
-      replicas   = 2
-      zones      = ["us-west2-b"]
+      type       = "n2-standard-2"
```

Comment:
What if we use `e2-micro` instead? `n2-standard-2` is about 15x the cost of `f1-micro` according to https://gcloud-compute.com/instances.html

`e2-micro` seems to be available in all regions as well.

Reply:
@jamilbk you can not reserve them, the whole purpose of this change is to have something that will be always available. I've switched them to `highcpu` variant with is about half the price.